model/project_ref.go

```diff
@@ -2326,19 +2327,19 @@ func (p *ProjectRef) GetActivationTimeForVariant(variant *BuildVariant) (time.Ti
 		}
 	}
 
-	return defaultRes, nil
+	return defaultTime, nil
 }
 
 // GetActivationTimeForTask returns the time at which this task should next be activated.
-// Temporarily takes in the task ID that prompted this query, for logging.
-func (p *ProjectRef) GetActivationTimeForTask(t *BuildVariantTaskUnit) (time.Time, error) {
-	defaultRes := time.Now()
+// Accepts a default time to ensure consistency across tasks. It's recommended to use the version
+// create time here rather than the task ingest time, since this can differ by task.
```

Comment:
Is it recommended or required? It seems like all the calls to this method use the version creation time rather than task ingest time anyways.

If it's always going to be the version creation time, could we call the parameter`versionCreationTime` or `versionCreatedAt`?